pyproject.toml

```diff
@@ -55,7 +56,23 @@ lint.select = [
     "Q",   # flake8-quotes
 ]
 lint.ignore = [
+    "D100", # docstring: public module
```

Comment:
Is anything left enabled? :wink:

Reply:
Ha!

With the following diff:
```diff
diff --git a/pyproject.toml b/pyproject.toml
index 842ff4e19..2645f0267 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -56,21 +56,6 @@ lint.select = [
     "Q",   # flake8-quotes
 ]
 lint.ignore = [
-    "D100", # docstring: public module
-    "D101", # docstring: public class required
-    "D102", # docstring: public method required
-    "D103", # docstring: public function required
-    "D107", # docstring: __init__ required
-    "D104", # docstring: public package required
-    "D105", # docstring: magic method required
-    "D200", # docstring: one line docstring shouldn't wrap
-    "D202", # docstring: blank line
-    "D205", # docstring: 1 blank line between initial and summary
-    "D209", # docstring: closing quotes -> separate line
-    "D400", # docstring: end with a period
-    "D401", # docstring: imperative mood
-    "D402", # docstring: docstring shouldn't start with func signature
-    "D403", # docstring: capitalized first line
     "E731", # Do not assign a `lambda` expression, use a `def`
 ]
 
```

I see:
```
Found 7269 errors.
[*] 542 fixable with the `--fix` option (1072 hidden fixes can be enabled with the `--unsafe-fixes` option).
```

Dropping the D20[2,5,9] suppressions is what I'd suggest next:
```
Found 1032 errors.
[*] 351 fixable with the `--fix` option 
```

then the `D40[0-3]`'s:
```
Found 1381 errors.
[*] 191 fixable with the `--fix` option
```

Maybe when I've got less changes in flight I (or someone else) can pick up the rest of it.